<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit</class>
 <widget class="QDialog" name="deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Срок размещения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="deposit_length"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>месяцев</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Процентная ставка</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="percent_deposit"/>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="capitalization">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Капитализация</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Ставка ЦБ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="CB_stavka"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_add_repelishment">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить пополнение</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>481</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Месяц</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="widhrawl_month">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Год</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="widhrawl_year">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Сумма</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="widhrawl_summ"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Полученные проценты</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="final_percent">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Налог</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="tax">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Сумма вклада к концу срока</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="final_deposit_summ">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_calculate_deposit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать депозит</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Сумма вклада</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="deposit_summ"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>481</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Периодичность выплат</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="deposit_periodicity">
      <item>
       <property name="text">
        <string>Ежемесячная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ежеквартальная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Полугодичная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ежегодная</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>В конце срока</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>481</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Месяц</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="repelishment_month">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Год</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="repelishment_year">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Сумма</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="repelishment_summ"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>46</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Снятия</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_widhrawl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить снятие</string>
   </property>
  </widget>
  <widget class="QComboBox" name="list_of_widhrawls">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_widhrawl">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Убрать снятие</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>77</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Пополнения</string>
   </property>
  </widget>
  <widget class="QComboBox" name="list_of_repelishment">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_repelishment">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Убрать пополнение</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
